ota-plus-web/app/reactapp/src/components/device/queue/MtuList.jsx

```diff
@@ -30,6 +30,11 @@ class MtuQueueList extends Component {
                         <ul className={"overview-panel__list" + (!devicesStore.multiTargetUpdates.length ? " empty" : "")}>
                             {_.map(devicesStore.multiTargetUpdates, (update, index) => {
                                 return _.map(update.targets, (target, serial) => {
+                                let itemEvents = devicesStore.deviceEvents.filter(el=> {
+                                    if ( el.payload.correlationId ) {
+                                        return el.payload.correlationId.search(update.updateId) >= 0

```

Comment:
Where does it filter events for ECU serial? Will need to test against device with secondary ecu.

Reply:
The test against secondary will be conducted when the info for secondary will be avaialble in `api/v1/auditor/update_reports` =>operationResults